.github/2.0/cookbooks/Executor.md

```diff
@@ -573,7 +579,7 @@ with Flow().\
 
 ---
 
-## Migration in Practice
+## Migrating from 1.x to 2.0 in Practice
```

Comment:
Should this section be removed completely? @hanxiao

Reply:
yes, i'm taking this branch now